androidClient/app/src/main/res/values/strings.xml

```diff
@@ -3,20 +3,37 @@
     <string name="app_name">Comrades</string>
     <string name="title_activity_sign_in">SignInActivity</string>
     <string name="title_activity_download">DownloadActivity</string>
-    <string name="server_client_id">666225132801-iklcdj36jau98rf2v44a0v1rnguioatd.apps.googleusercontent.com</string>
+    <string name="server_client_id">
+        666225132801-iklcdj36jau98rf2v44a0v1rnguioatd.apps.googleusercontent.com
+    </string>
     <string name="client_secret">NGfzk-7sJilNmFla6TZy8WrL</string>
-    <string name="redirect_url">https://balmy-component-204213.firebaseapp.com/__/auth/handler</string>
-    <string name="drive_upload_url">https://www.googleapis.com/upload/drive/v3/files?uploadType=media</string>
+    <string name="redirect_url">
+        https://balmy-component-204213.firebaseapp.com/__/auth/handler
+    </string>
+    <string name="drive_upload_url">
+        https://www.googleapis.com/upload/drive/v3/files?uploadType=media</string>
     <string name="drive_api_base_url">https://www.googleapis.com/drive/v3/files/</string>
     <string name="action_search">Search App</string>
     <string name="login_to_add_course">Please login to add course.</string>
+    <string name="login_to_add_file">Please login to add material.</string>
+    <string name="login_to_download_file">Please login to download material.</string>
+    <string name="storage_permission_needed">Storage permission needed</string>
     <string name="login">Log In</string>
-    <string name="sign_in_failes">Sign in failed</string>
+    <string name="allow">Allow</string>
+    <string name="sign_in_fails">Sign in failed</string>
     <string name="retry">Retry</string>
     <string name="bits_only_email_allowed">Only BITS email is allowed</string>
     <string name="sign_out">Sign Out</string>
     <string name="add_course">Add Course</string>
+    <string name="add_file">Add File</string>
     <string name="cancel">Cancel</string>
     <string name="course_name">course name</string>
-    <string name="warn_empty_course_name">Course came should now be empty</string>
+    <string name="warn_empty_course_name">Course name should not be empty</string>
+    <string name="warn_empty_file_name">File name cannot be empty</string>
+    <string name="warn_select_file">Please select a file</string>
+    <string name="DOWNLOAD_DIRECTORY">/Comrades/</string>
```

Comment:
this doesn't belong here

Reply:
didn't get this.All these are strings used across the project.so, they should be here.do you mean in caps?